modules/dxp/apps/vldap/vldap-server-test/src/test/java/com/liferay/vldap/server/internal/portal/security/auth/SambaAuthTest.java

```diff
@@ -28,19 +29,22 @@
 
 import org.junit.Assert;
 import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Rule;
 import org.junit.Test;
-import org.junit.runner.RunWith;
 
 import org.mockito.Mockito;
 
-import org.powermock.modules.junit4.PowerMockRunner;
-
 /**
  * @author Jonathan McCann
  */
-@RunWith(PowerMockRunner.class)
 public class SambaAuthTest extends BaseVLDAPTestCase {
```

Comment:
@JanisZhang These tests are extending `BaseVLDAPTestCase` which still uses PowerMock. Could you also change that class and include it in this PR?

Reply:
@dantewang  Hi Dante, The reason I didn't include `BaseVLDAPTestCase` in this PR is that it still have one class `DirectoryTreeTest` (powermock **LdapUtil**)  extends BaseVLDAPTestCase. I still trying to find a way to replace LdapUtil in `DirectoryTreeTest` yet...

Reply:
@JanisZhang OK, let's wait for this to be done. If you can't find a way please let me know so we can look into it together. I'm closing this one for now.